src/Mod/Arch/importIFC.py

```diff
@@ -20,6 +20,7 @@
 #*   USA                                                                   *
 #*                                                                         *
 #***************************************************************************
+# Modified Amritpal Singh <amrit3701@gmail.com> on 02-01-2018
```

Comment:
this line is not needed IMHO. This is exactly what is git for.

Reply:
Okay, I will remove this line in my next commit. 

@berndhahnebach @yorikvanhavre 
Travis ci test fails when PYTHON_MAJOR_VERSION=3. After seeing job log, I found that below Fem test fail.  

```
======================================================================
ERROR: test_pyimport_all_FEM_modules (femtest.testfemcommon.FemTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/Mod/Fem/femtest/testfemcommon.py", line 433, in test_pyimport_all_FEM_modules
    __import__('{0}'.format(mod))  # to get an error message what was going wrong
  File "/usr/local/Mod/Fem/feminout/importFenicsMesh.py", line 40, in <module>
    from . import writeFenicsXML
  File "/usr/local/Mod/Fem/feminout/writeFenicsXML.py", line 33, in <module>
    from importToolsFem import get_FemMeshObjectDimension, get_FemMeshObjectElementTypes, get_MaxDimElementFromList
ImportError: No module named 'importToolsFem'

----------------------------------------------------------------------
```
Thanks